### PR TITLE
11 Connection entity class

### LINKED ISSUE #11: Connection entity class
**Description of the feature to be developed:**
Make the entity class "Connection" and add the variable fields found in the class diagram to it.

### PR DESCRIPTION
**Pull request by Teyson**
Closes #11

## Description
Added the connection entity class and populated it with fields and methods

## Checklist:
- [x] My code follows the code style of this project.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests pass.
- [x] Sonar check has passed
- [x] No codesmells or bugs

### CODE REVIEW

**Review by Kaszz — COMMENTED** [review]
Muligvis skal ID ikke gives i constructore, da det forhåbentligt er muligt at give i databasen

File: BrewMesModulesParent/common/src/test/java/com/brewmes/common/entities/ConnectionTest.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ConnectionTest {
+    Connection connection;
+
+    @BeforeEach
+    void setUp() {
+        this.connection = new Connection(1,"128.0.0.1", "testMachine");
```
Comment (Kaszz) [inline BrewMesModulesParent/common/src/test/java/com/brewmes/common/entities/ConnectionTest.java:14]: Overvej at benytte 420.69.69.69

**Review by Wylardt — APPROVED** [review]

**Review by Kaszz — APPROVED** [review]
🐸

**Review by Lawst96 — APPROVED** [review]

File: BrewMesModulesParent/common/src/main/java/com/brewmes/common/entities/Connection.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
+package com.brewmes.common.entities;
+
+public class Connection {
+    private int id;
+    private String ip;
+    private String name;
+
+    public Connection(int id, String ip, String name) {
```
Comment (dottore1) [inline BrewMesModulesParent/common/src/main/java/com/brewmes/common/entities/Connection.java:8]: Bør dette være et String id i stedet for int?

Comment (Teyson) [reply BrewMesModulesParent/common/src/main/java/com/brewmes/common/entities/Connection.java]: Tjek igen

**Review by dottore1 — APPROVED** [review]

**Review by Kechr — APPROVED** [review]
👎🏽

**Pull request merged**